.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
If I'm not mistaken this file can be removed.

Reply:
you're right, I'll remove it